src/Common/dataAccess/readCollection.ts

```diff
@@ -22,3 +26,51 @@ export async function readCollection(databaseId: string, collectionId: string):
   clearMessage();
   return collection;
 }
+
+export async function getMongoCollectionIndexTransformationProgress(
```

Comment:
Why are we reading this via the SDK? Is this not exposed via RP?

Reply:
I sorta understand what is going on now reading the rest of the code. How are connection string users supported? Can they not update their indexes using this new component.

Reply:
RP doesn't support getting the "index progress" header so this has to be done through the client. Is there anyway we can fetch the cosmos collection using a connectionstring? if not, connectionstring users should not even be able to update their collection in way right now (like ttl, analytical ttl). How does that take place for connection string users? I can use the same pattern.

Reply:
synced offline. If user is not using AAD, will show a prompt that they will have to sign in to see the indexing tab.